README.md

```diff
@@ -107,6 +107,13 @@ Host Docker storage drivers of `overlay` and `aufs` are supported automatically.
 
 For other make commands, see `make help`.
 
+## DC/OS Login
+
```

Comment:
please add a line about this being specific to Enterprise DC/OS installs and that OSS DC/OS uses OAuth for login.